core/platform/auth/firebase_auth_services_test.py

```diff
@@ -54,50 +57,69 @@ class Test(test_utils.TestBase):
 
             def setUp(self):
                 super(Test, self).setUp()
-                self._uninstall_stub = FirebaseAdminSdkStub.install(self)
+                self.sdk_stub = FirebaseAdminSdkStub(self)
```

Comment:
Leaning a bit towards self.firebase_sdk_stub since this sounds really generic. But optional.

Reply:
I've changed the docstring, but I think it's fine for this file to use the shorter name since Firebase is already implied, and because this is only an example not a rule/standard.

Reply:
I disagree with the reasoning, FWIW -- it will become a rule/standard by fiat because it's what people will copy from. I've seen this happen multiple times already in other cases.